packages/terra-demographics-banner/tests/nightwatch/demographics-banner-spec.js

```diff
@@ -0,0 +1,95 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
That I can do. https://github.com/cerner/terra-core/pull/280/commits/4d4cd61c658a3bac2f114f57b33370c6427f1924